Mathlib/Data/QPF/Univariate/Basic.lean

```diff
@@ -167,14 +167,14 @@ set_option linter.uppercaseLean3 false in
 #align qpf.recF QPF.recF
 
 theorem recF_eq {α : Type _} (g : F α → α) (x : q.P.W) :
```

Comment:
With your changes, can these `Type _`s be turned into `Type*`?

Reply:
We cannot because the universe of `F` is fixed.